building.html

```diff
@@ -151,11 +151,49 @@ <h3 id="building-windows">Building on Windows</h3>
       </ol>
 
 
-      <h3 id="building-nix">Building on macOS &amp; Linux</h3>
+      <h3 id="building-mac">Building on macOS</h3>
 
-      <p>Linux and macOS are <strong>not officially supported targets</strong>, and the game client does not compile. However, several of the tools can be built and run.</p>
+      <p>Compiling on macOS requires a minimum of macOS 11 and the Xcode build system. Xcode is available as a free download from the <a href="https://apps.apple.com/us/app/xcode/id497799835?mt=12/">App Store</a>.</p>
 
-      <p>You will need to ensure all the required dependencies are available. On macOS, a Homebrew bundle file is included in the repository, which can be installed by running <code>brew bundle</code>. You can also use vcpkg on these platforms for automatically managing dependencies.</p>
+      <p>We strongly encourage using vcpkg to automatically build dependencies; however, this requires some tools be manually installed in advance. The required tools are CMake, nasm, and pkg-config, and can be installed using common package managers such as Homebrew or MacPorts.</p>
```

Comment:
```suggestion
      <p>We strongly encourage using vcpkg to automatically build dependencies; however, this requires some tools be manually installed in advance. The required tools are CMake, nasm, and pkg-config. They can be installed using common package managers such as Homebrew or MacPorts.</p>
```

The comma before the second `and` is not quite correct, and that structure is a little awkward, IMO. It would be better to separate that into two sentences.